package.json

```diff
@@ -1,23 +1,31 @@
 {
-  "name":        "sleep",
-  "version":     "3.0.1",
-  "main":        "index.js",
+  "name": "sleep",
+  "version": "3.0.1",
+  "main": "index.js",
   "description": "Add sleep() and usleep() to nodejs",
-  "homepage":    "http://github.com/ErikDubbelboer/node-sleep",
-  "author":      "Erik Dubbelboer <erik@dubbelboer.com>",
-  "license":     "MIT",
-  "engines" : { "node" : ">=0.4.0" },
+  "homepage": "http://github.com/ErikDubbelboer/node-sleep",
+  "author": "Erik Dubbelboer <erik@dubbelboer.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "mocha"
+  },
+  "engines": {
+    "node": ">=0.4.0"
+  },
   "keywords": [
     "sleep",
     "usleep"
   ],
   "repository": {
     "type": "git",
-    "url":  "https://github.com/ErikDubbelboer/node-sleep.git"
+    "url": "https://github.com/ErikDubbelboer/node-sleep.git"
   },
   "dependencies": {
     "nan": ">=2.0.0"
   },
+  "devDependencies": {
+    "mocha": "^3.0.2"
+  },
```

Comment:
This and the `scripts` attribute above are the only important changes to `package.json`. The rest were just automatically done by `npm install -D mocha`. Let me know if you'd like me to revert the rest.